packages/atomic/src/pages/examples/ipx.html

```diff
@@ -231,8 +242,13 @@
             <atomic-notifications></atomic-notifications>
           </atomic-layout-section>
           <atomic-layout-section section="results">
-            <atomic-smart-snippet></atomic-smart-snippet>
-            <atomic-smart-snippet-suggestions></atomic-smart-snippet-suggestions>
+            <atomic-generated-answer></atomic-generated-answer>
+            <atomic-smart-snippet>
+              <a slot="attributes" target="_blank"></a>
+            </atomic-smart-snippet>
+            <atomic-smart-snippet-suggestions>
+              <a slot="attributes" target="_blank"></a>
+            </atomic-smart-snippet-suggestions>
```

Comment:
Don't forget to add [noopener](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) to avoid potential vulnerabilities in old browsers (just in case).
```suggestion
            <atomic-generated-answer></atomic-generated-answer>
            <atomic-smart-snippet>
              <a slot="attributes" target="_blank" rel="noopener"></a>
            </atomic-smart-snippet>
            <atomic-smart-snippet-suggestions>
              <a slot="attributes" target="_blank" rel="noopener"></a>
            </atomic-smart-snippet-suggestions>
```